### DISCUSSION TITLE
export any imported sliced model to bmp or (long) pdf

### DISCUSSION THREAD

**Discussion by jeedee23** [sn4k3/UVtools#517] [Q&A] [answered]
EDIT!!! What a stupid question I asked: just under 'actions: export layers as image...'
But what about all the settings, changes, etc. Can we export them to an xml or csv or whatever file?

Maybe one of the export options is already giving this option. What I want to do is export a sliced model into my own software and then print it with my own self-made printer, of course using a lot of the possibilities UVtools gives.
I looked through the source, but, lazy as I am, I thought 'why not ask?''.

Thanks for the response!

**Comment by sn4k3** [comment, marked as answer]
`File -> Extract File Contents` will include settings
You can also use [UVtoolsCmd](https://github.com/sn4k3/UVtools/#uvtoolscmd-console-executable)

**Reply by jeedee23** [reply]
Great! This is really really really a good app!
